generators/chipyard/src/main/scala/config/RocketConfigs.scala

```diff
@@ -23,7 +23,7 @@ class TinyRocketConfig extends Config(
 
 // DOC include start: FFTRocketConfig
 class FFTRocketConfig extends Config(
-  new fftgenerator.WithFFTGenerator(baseAddr=0x2000, numPoints=8, width=16, decPt=8) ++ // add 8-point mmio fft at 0x2000 with 16bit fixed-point numbers.
+  new fftgenerator.WithFFTGenerator(numPoints=8, width=16, decPt=8) ++ // add 8-point mmio fft at 0x2000 with 16bit fixed-point numbers.
```

Comment:
Comment needs to be updated. However, why was this address removed in favor of it being @ `0x2400`?

Reply:
Making all the MMIO devices use different addresses makes it easier to generate and test 1 config that contains many devices.

I updated the SW tests as well for these.